defaults/main.yml

```diff
@@ -116,6 +116,16 @@ checkmk_agent__user_home: '/var/lib/nagios'
 # SSH Public key for logging in with SSH user.
 checkmk_agent__user_key: ''
 
+
+# .. envvar:: checkmk_agent__authorized_keys__dependent_list
+#
+# Authorized key configuration for the ``debops.authorized_keys`` role.
+checkmk_agent__authorized_keys__dependent_list:
+  - name: '{{ checkmk_agent__ssh_user }}'
+    ssh_keys: [ '{{ checkmk_agent__user_key }} if {{ checkmk_agent__user_key|d() }} else {{ hostvars[groups["debops_service_checkmk_server"][0]].ansible_local.checkmk_server.ssh_publickey|d("") }}' ]
```

Comment:
@ganto I see everything. :-) And sure, if a role has support for use as a dependency, I'm fine with it. Otherwise I wouldn't add variables like `*_dependent_*`.